src/AzSK.ADO/Framework/Configurations/SVT/ADO/ADO.Organization.json

```diff
@@ -346,14 +346,14 @@
       "Enabled": true
     },
     {
-      "ControlID": "ADO_Organization_AuthZ_Limit_Non_Release_Pipeline_Access",
+      "ControlID": "ADO_Organization_AuthZ_Limit_Non_Release_Pipeline_Scope",
       "Description": "Limit scope of access for non-release pipelines to the current project.",
       "Id": "Organization320",
       "ControlSeverity": "Medium",
       "Automated": "Yes",
       "MethodName": "CheckJobAuthZScope",
-      "Rationale": "If pipelines use project collection level tokens, a vulnerability in components used by one project can be leveraged by an attacker to attack all other projects. This is also in keeping with the principle of least privilege.",
-      "Recommendation": "Go to Organization Settings --> Pipelines --> Settings --> Enable 'Limit job authorization scope to current project for non-release pipelines'.",
+      "Rationale": "If this setting is disabled, a hacker can build a pipeline in any misconfigured project to gain access to most resources in an organization.This is also in keeping with the principle of least privilege.",
```

Comment:
Rationale should be specific, Should be something like below :
If the authorization scope of non release pipeline is allowed to multiple projects, an attacker can build a pipeline in any misconfigured project to gain access to most resources in an organization.This is also in keeping with the principle of least privilege.

Also please get it reviewed with manish